control-plane/csi-controller/src/controller.rs

```diff
@@ -212,6 +212,7 @@ impl rpc::csi::controller_server::Controller for CsiControllerSvc {
                 )))
             }
         };
+        tracing::Span::current().record("volume.uuid", &volume_uuid.as_str());
```

Comment:
Just out of my curiosity: why we use tracing::Span: only in create_volume() ?

Reply:
because the volume uuid is not trivially accessible from the request so it's set to Empty. Then here we already have the uuid so can finally record it in the span.